src/Drupal/ContentBlockTrait.php

```
@@ -163,6 +166,35 @@ public function contentBlockCreate(string $type, TableNode $content_block_table)
+   * @code
+   * Given basic content blocks with fields:
+   *   | field  | block_1               | block_2               |
+   *   | info   | [TEST] Block 1        | [TEST] Block 2        |
+   *   | body   | First block content   | Second block content  |
+   *   | status | 1                     | 1                     |
+   * @endcode
+   */
+  public function contentBlockCreateWithFields(string $type, TableNode $table): void {
+    $entities = $this->utilityTransposeVerticalTable($table);
+
+    foreach ($entities as $entity_data) {
```

PHPMD (2.15.0)

The variable $entity_data is not named in camelCase. (undefined)

(CamelCaseVariableName)